views/src/app/Auth/components/comments/comments.js

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import { Container, Row, Col, Button, FormGroup, FormControl } from 'react-bootstrap';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import './comments.css';
+
+class CommentsContainer extends Component {
+    constructor(props) {
+        super(props);
+        this.state = { comments: [] };
+    }
+
+    getComments = async () => {
+        try {
+            let res = await axios.get(`http://localhost:3001/api/posts/byid/${this.props.postItem}`);
+            if (res.status === 200) {
+                this.setState({ comments: res.data.comments });
+            } else {
+                console.log("FAIL get comments:", res.status);
+            }
+        } catch (err) {
+            console.log("FAIL get comments: ", err);
+        }
+    }
+
+    postComment = async () => {
+        let body = this.commentEditor.value;
+        try {
+            let res = await axios.post('http://localhost:3001/api/comments/', { body, postItem: this.props.postItem });
+            if (res.status === 201) {
+                console.log("Successfully creating a comment");
+                console.log(res.data);
+            } else {
```

Comment:
When does this get sent to the FE?

Reply:
I'm not sure I got this. Did you mean displaying this comment to the user? Definitely gonna remove the second `console.log`.